lib/dependabot/shared_helpers.rb

```diff
@@ -62,34 +63,57 @@ def self.in_a_forked_process
     end
 
     class HelperSubprocessFailed < StandardError
-      def initialize(message, command)
+      def initialize(message, error_context:)
         super(message)
-        @command = command
+        @error_context = error_context
+        @command = error_context[:command]
       end
 
       def raven_context
-        { fingerprint: [@command] }
+        { fingerprint: [@command], extra: error_context }
```

Comment:
Wouldn't you need a `attr_reader :error_context` or this:

```suggestion
        { fingerprint: [@command], extra: @error_context }
```

?